samples/testing/operations/src/Test_SWAP.qs

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+/// # Sample
+/// Multi File Testing Project
+///
+/// # Description
+/// This code builds upon the concepts explained in the README file,
+/// demonstrating how to organize Q# code into multiple files for testing.
+
+namespace Test_SWAP {
```

Comment:
The previous comment chain disappeared, so here's a new one: I agree that it's good to name namespaces with names distinct from the operations defined in them, but it's also good not to put non-test logic in namespace named "test". We can use "CustomOperation" or "CustomSWAP" to satisfy both